apps/blockchain/starknet/src/tests/bridge_t.cairo

```diff
@@ -1080,4 +1080,138 @@ mod tests {
         stop_prank(CheatTarget::One(bridge_address));
     }
     
+    #[test]
+    fn enable_white_list() {
+        let erc721b_contract_class = declare("erc721_bridgeable");
+
+        let BRIDGE_ADMIN = starknet::contract_address_const::<'starklane'>();
+        let BRIDGE_L1 = EthAddress { address: 'starklane_l1' };
+
+        let bridge_address = deploy_starklane(BRIDGE_ADMIN, BRIDGE_L1, erc721b_contract_class.class_hash);
+        let bridge = IStarklaneDispatcher { contract_address: bridge_address };
+
+        assert_eq!(bridge.is_white_list_enabled(), false);
+
+        let mut spy = spy_events(SpyOn::One(bridge_address));
+        start_prank(CheatTarget::One(bridge_address), BRIDGE_ADMIN);
+        bridge.enable_white_list(true);
+        stop_prank(CheatTarget::One(bridge_address));
+        spy.assert_emitted(@array![
+            (
+                bridge_address,
+                bridge::Event::WhiteListEnabled(
+                    bridge::WhiteListEnabled {
+                        enabled: true,
+                    }
+                )
+            )
+        ]);
+
+        assert_eq!(bridge.is_white_list_enabled(), true);
+    }
+
+    #[test]
+    fn set_bridge_l1_addr() {
+        let erc721b_contract_class = declare("erc721_bridgeable");
+
+        let BRIDGE_ADMIN = starknet::contract_address_const::<'starklane'>();
+        let BRIDGE_L1 = EthAddress { address: 'starklane_l1' };
+        let NEW_BRIDGE_L1 = EthAddress { address: 'new_starklane_l1' };
+
+        let bridge_address = deploy_starklane(BRIDGE_ADMIN, BRIDGE_L1, erc721b_contract_class.class_hash);
+        let bridge = IStarklaneDispatcher { contract_address: bridge_address };
+
+        assert_eq!(bridge.get_bridge_l1_addr(), BRIDGE_L1);
+
+        let mut spy = spy_events(SpyOn::One(bridge_address));
+        start_prank(CheatTarget::One(bridge_address), BRIDGE_ADMIN);
+        bridge.set_bridge_l1_addr(NEW_BRIDGE_L1);
+        stop_prank(CheatTarget::One(bridge_address));
+        spy.assert_emitted(@array![
+            (
+                bridge_address,
+                bridge::Event::BridgeL1AddressUpdated(
+                    bridge::BridgeL1AddressUpdated {
+                        address: NEW_BRIDGE_L1,
+                    }
+                )
+            )
+        ]);
+
+        assert_eq!(bridge.get_bridge_l1_addr(), NEW_BRIDGE_L1);
+    }
+
+    #[test]
+    fn set_erc721_class_hash() {
+        let erc721b_contract_class = declare("erc721_bridgeable");
+
+        let BRIDGE_ADMIN = starknet::contract_address_const::<'starklane'>();
+        let BRIDGE_L1 = EthAddress { address: 'starklane_l1' };
+        let CLASS_HASH_ZERO = class_hash_const::<0>();
```

Comment:
Please use a classhash different than 0 since there is another issue which prevent classhash to be 0.